homeassistant/components/plexamp/services.py

```diff
@@ -0,0 +1,374 @@
+"""Handles communication with Plexamp services."""
```

Comment:
`services.py` is reserved for HA services. This looks like it is the downstream lib